### ISSUE TITLE
sync command - TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by lovasoa** [openode-io/openode-cli#17]
Similarly to #15, I recently got the following error while running openode sync : 

```
TypeError: Cannot read property 'replace' of undefined
    at normalizeApiUrl (/private/tmp/wbo/node_modules/openode/modules/cliConfs.js:9:14)
    at Object.determineClosestEndpoint (/private/tmp/wbo/node_modules/openode/modules/cliConfs.js:19:19)
    at async Object.prepareAuthenticatedCommand (/private/tmp/wbo/node_modules/openode/modules/main.js:33:5)
    at async prepareAuth (/private/tmp/wbo/node_modules/openode/index.js:38:17)
    at async Command.<anonymous> (/private/tmp/wbo/node_modules/openode/index.js:186:25)
⠋ run command err
Error: Unable to process the location undefined
    at Object.locations2Process (/private/tmp/wbo/node_modules/openode/modules/locations.js:41:11)
    at async processAllLocations (/private/tmp/wbo/node_modules/openode/index.js:56:25)
    at async progress (/private/tmp/wbo/node_modules/openode/index.js:656:16)
    at async runCommand (/private/tmp/wbo/node_modules/openode/index.js:20:18)
    at async Command.<anonymous> (/private/tmp/wbo/node_modules/openode/index.js:192:7)
```

Re-running the command did not reproduce the error.

**Comment by martinlevesque** [comment]
Thanks for reporting it, will look at it as soon as possible.

**Comment by martinlevesque** [comment]
fixed in the new CLI version (2.0.16), npm i -g openode to upgrade
